colorbleed/lib.py

```diff
@@ -257,16 +257,66 @@ def get_project_fps():
 
     Returns:
         int, float
+
+    """
+
+    data = get_project_data()
+    fps = data.get("fps", 25.0)
+
+    return fps
+
+
+def get_project_data():
+    """Get the data of the current project
+
+    The data of the project can contain things like:
+        resolution
+        fps
+        renderer
+
+    Returns:
+        dict:
+
     """
 
     project_name = io.active_project()
     project = io.find_one({"name": project_name,
                            "type": "project"},
-                          projection={"config": True})
+                          projection={"data": True})
+
+    data = project.get("data", {})
+
+    return data
+
 
-    config = project.get("config", None)
-    assert config, "This is a bug"
+def get_asset_data(asset=None):
+    """Get the data from the current asset
+
+    Args:
+        asset(str, Optional): name of the asset, eg:
+
+    Returns:
+        dict
+    """
+    
+    asset_name = asset or avalon.api.Session["AVALON_ASSET"]
+    document = io.find_one({"name": asset_name,
+                            "type": "asset"})
+
+    data = document.get("data", {})
+
+    return data
+
+
+def get_asset_fps():
```

Comment:
I think this method is now redundant with [this line](https://github.com/Colorbleed/colorbleed-config/pull/157/files#diff-b9b06b643a808ffdecb81fd665eec054R1454) so let's remove this function.

Reply:
This would also count for project fps, right?

Reply:
Now that you mention it - I suppose so?

Reply:
You're correct I think. It should also "behave" correctly for the "on_open", "on_save" that it knows to grab it from asset first, then project.

Well spotted.